tests/System.IO.Pipelines.Tests/PipeLengthTests.cs

```diff
@@ -106,7 +102,7 @@ public void LengthNotChangeAfterReadAdvanceExamine()
         [Fact]
         public void ByteByByteTest()
         {
-            WritableBuffer writableBuffer;
+            WritableBuffer writableBuffer = default(WritableBuffer);
```

Comment:
Compiler just randomly started to error because of uninitialized variable on this line.